src/main.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.tsx";
+import "./styles/main.css"
```

Comment:
Could u please add this folder in the configs (tsconfig + vite.config) so we can have absolute path

```suggestion
import "Styles/main.css"
```

Reply:
sure